pkg/reconciler/kubernetes/tektonresult/filter.go

```diff
@@ -26,10 +26,11 @@ const (
 	servicePostgresDB = "tekton-results-postgres-service"
 )
 
-func (r *Reconciler) filterExternalDB(tr *v1alpha1.TektonResult) {
+func (r *Reconciler) filterExternalDB(tr *v1alpha1.TektonResult, manifest mf.Manifest) mf.Manifest {
```

Comment:
why we are passing this, can we take it from reconcile?

Reply:
We were passing it because we didn't want to modify r.Manifest.